### DISCUSSION TITLE
Is there a way to display the rows and columns when I resize the window?

### DISCUSSION THREAD

**Discussion by sfsam** [kovidgoyal/kitty#6357] [Q&A] [answered]
Some terminal emulators (Terminal.app, iTerm2) display the number of rows and columns in the title bar when the window is resized. Is there a way to do this with kitty? Thank you.

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.resize_draw_strategy

On Sun, Jun 11, 2023 at 04:31:43PM -0700, Sanjay wrote:
> Some terminal emulators (Terminal.app, iTerm2) display the number of rows and columns in the title bar when the window is resized. Is there a way to do this with kitty? Thank you.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6357
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
